pkg/rpcclient/wsclient_test.go

```diff
@@ -574,6 +575,24 @@ func TestWSFilteredSubscriptions(t *testing.T) {
 				require.Equal(t, util.Uint256{1, 2, 3}, *filt.Container)
 			},
 		},
+		{"notary requests",
+			func(t *testing.T, wsc *WSClient) {
+				sender := util.Uint160{1, 2, 3, 4, 5}
+				signer := util.Uint160{0, 42}
+				mempoolType, err := mempoolevent.GetEventTypeFromString("added")
+				require.NoError(t, err)
+				_, err = wsc.ReceiveNotaryRequests(&neorpc.NotaryRequestFilter{Type: &mempoolType, Signer: &signer, Sender: &sender}, make(chan *result.NotaryRequestEvent))
+				require.NoError(t, err)
+			},
+			func(t *testing.T, p *params.Params) {
+				param := p.Value(1)
+				filt := new(neorpc.NotaryRequestFilter)
+				require.NoError(t, json.Unmarshal(param.RawMessage, filt))
+				require.Equal(t, util.Uint160{1, 2, 3, 4, 5}, *filt.Sender)
+				require.Equal(t, util.Uint160{0, 42}, *filt.Signer)
+				require.Equal(t, "added", filt.Type.String())
+			},
+		},
```

Comment:
Please, pay attention to the overall `TestWSFilteredSubscriptions` test structure, notary filter test should be similar to the rest of the cases. Here you need to have four tests: `notary request sender`, `notary request signer`, `notary request type` and `notary request sender, signer and type`. Check out the way how transaction-related tests are implemented.

Reply:
fixed